doc: document the way to load samples (#23)
openfoodfacts/taxonomy-editor: doc/how-to-guides/develop-with-neo4j.md

[^vol_name]: (it may have another name, look at what `docker volume list|grep neo4j-data`) gives you.

> ## importing data

Reviewer: ```suggestion
## Importing data into Neo4J database
```

> ### import sample data

Reviewer: ```suggestion
### Import sample data
```

We have a `backend/sample/test-neo4j.json` file to load sample data in the database for developers.

> To load it:

Author: git clone of what ? 
- The project --> of course, but it''s the readme that should explain that
- A neo4j repository --> no

> To load it:

Reviewer: Ah okay, I meant for the project only. Thanks for the clarification!

> * ensure your nep4j database is launched:

Reviewer 2: ```suggestion
* ensure your neo4j database is launched:
```

  ```bash
  docker-compose up -d neo4j
  ```
* create you local env if you do not have one yet:
  ```bash
  python3 -m venv .venv
  ```
* activate your local env, if needed:
  ```bash
  . .venv/bin/activate
  ```
* if needed, install neo4j driver:
  ```bash
  pip install neo4j
  ```
* finally load the data:
  ```bash
  cd backend/sample
  python3 load.py --reset test-neo4j.json
  ```

  note that if you want to see more options, you can use `--help`
* enjoy it in the neo4j browser at http://localhost:7474
  (hint: after connecting the database, on the right click on the "*" node types to have a beautiful graph)

> ### from a generic file

Reviewer: ```suggestion
### From a generic file
```

> If you want to import data, the file needs to be placed inside the `neo4j/import` folder.

Reviewer: ```suggestion
If you want to import data, the file needs to be placed inside the `neo4j/import` folder.
This jsonl file must follow all specifications mentioned in [this file](https://github.com/openfoodfacts/taxonomy-editor/blob/main/doc/references/taxonomy-parser.md)
```

Author: No, this was paragraph is just about doing a generic import in Neo4J. I think it was forth explaining (because of the docker settings that might be less known to some people). But it's not about taxonomy specifically here. Maybe I should be more explicit ?

```suggestion
Note that if you want to import some other data, (eg for a specific test of Neo4J capabilities), 
the file needs to be placed inside the `neo4j/import` folder.
```

[^vol_name]: (it may have another name, look at what `docker volume list|grep neo4j-data`) gives you.

## Importing data into Neo4J database

### Import sample data

We have a `backend/sample/test-neo4j.json` file to load sample data in the database for developers.

To load it:

* ensure your neo4j database is launched:
  ```bash
  docker-compose up -d neo4j
  ```
* create you local env if you do not have one yet:
  ```bash
  python3 -m venv .venv
  ```
* activate your local env, if needed:
  ```bash
  . .venv/bin/activate
  ```
* if needed, install neo4j driver:
  ```bash
  pip install neo4j
  ```
* finally load the data:
  ```bash
  cd backend/sample
  python3 load.py --reset test-neo4j.json
  ```

  note that if you want to see more options, you can use `--help`
* enjoy it in the neo4j browser at http://localhost:7474

>   (hint: after connecting the database, on the right click on the "*" node types to have a beautiful graph)

Reviewer: ```suggestion
  (Hint: After connecting the database, on the leftclick on the "*" node types to visualize a beautiful graph of the loaded taxonomy)
```
